models/examples/e_05_contract.py

```diff
@@ -43,16 +52,51 @@ def run(self, _) -> ExampleModelOutput:
 
         output.log("You can get events by creating filters. To get all vested accounts, "
                    "we can query \"VestingScheduleAdded\" events.")
-        vesting_added_events = contract.events.VestingScheduleAdded.createFilter(
-            fromBlock=0,
-            toBlock=self.context.block_number
-        ).get_all_entries()
-
-        output.log_io(input="""
-vesting_added_events = contract.events.VestingScheduleAdded.createFilter(
-            fromBlock=0,
-            toBlock=self.context.block_number
-        ).get_all_entries()""", output=vesting_added_events)
+        try:
+            vesting_added_events = contract.events.VestingScheduleAdded.createFilter(
+                fromBlock=0,
+                toBlock=self.context.block_number
+            ).get_all_entries()
+
+            output.log_io(input="""
+    vesting_added_events = contract.events.VestingScheduleAdded.createFilter(
```

Comment:
soon we'll be able to do this via the views, rather than calling web3 events

Reply:
yeah. I read what Peter wrote about the views.